x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/actions_panel/actions_panel.tsx

```diff
@@ -61,7 +61,11 @@ export const ActionsPanel: FC<Props> = ({ indexPattern }) => {
         </EuiText>
         <EuiSpacer size="m" />
         <EuiFlexGroup gutterSize="l" responsive={true} wrap={true}>
-          <DataRecognizer indexPattern={indexPattern} results={recognizerResults} />
+          <DataRecognizer
+            indexPattern={indexPattern}
+            savedSearch={null}
```

Comment:
question: can it be a default value in `DataRecognizer ` component?

Reply:
`DataRecognizer` is an old js class react component so i think it'd have to be done like this in its constructor:
`this.savedSearch = props.savedSearch !== undefined ? props.savedSearch : null;`